src/components/Header/Header.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
+import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
+import { ReactComponent as DopplerLogo } from '../../doppler-logo.svg';
+
+class Header extends Component {
+
+  constructor() {
+    super();
```

Comment:
if I am not wrong, it is a good practice using _props_ parameter, something like:

```
  constructor(props) {
    super(props);
```

Reply:
i didn't know. good!

Reply:
👁 I am not sure, please research before paying attention to me opinion.